coalib/processes/Processing.py

```diff
@@ -138,7 +141,8 @@ def autoapply_actions(results,
                 not_processed_results.append(result)
                 continue
 
-        if not action.is_applicable(result, file_dict, file_diff_dict):
+        if ((not action.is_applicable(result, file_dict, file_diff_dict)) and
+                (not flag)):
```

Comment:
you can't just change this condition, this changes the behaviour because line 149 isn't executed if the warning isn't shown but it's vital to run that line even if the warning is omitted